core/templates/dev/head/pages/exploration_player/PlayerServices.js

```diff
@@ -369,9 +374,19 @@ oppia.factory('oppiaPlayerService', [
           exploration.isInteractionInline(oldStateName));
 
         if (!_editorPreviewMode) {
+          var isFirstHit = Boolean(visitedStateNames.indexOf(
+            newStateName) === -1);
           StatsReportingService.recordStateTransition(
             oldStateName, newStateName, answer,
-            LearnerParamsService.getAllParams());
+            LearnerParamsService.getAllParams(), isFirstHit);
+          visitedStateNames.push(newStateName);
+
+          if (oldStateName == exploration.initStateName && (
```

Comment:
===

Reply:
Also why the check for state name? If the first question has been designed to be too hard then that's an issue we should surface, right? (I.e. the learner has started, but has difficulty getting past the first card.)

Or did I misunderstand our discussion and we're only counting an "actual start" once the learner has advanced at least one card? That's fine with me too, it's just different from what I was thinking.

/cc @LanJosh

Reply:
@seanlip I thought we decided to go with advancement of just one card when we discussed about migration of existing events to calculate actual start counts.

Reply:
Ah, yes. OK, I'm fine with that; we'll be accounting for the other case in the "bounce rate" anyway.

Please ignore comments related to this, then.